types/block.go

```diff
@@ -44,8 +47,7 @@ type Block struct {
 
 	Header     `json:"header"`
 	Data       `json:"data"`
-	Evidence   EvidenceData `json:"evidence"`
-	LastCommit *Commit      `json:"last_commit"`
+	LastCommit *Commit `json:"last_commit"`
```

Comment:
note that the data availability header is not added to the block. Instead, we can simply generate the DAH and data hash, then discard it. 

If we want to save it for whatever reason, then we can add it seperately to `BlockMeta`

ref #512